open-api/rest-catalog-open-api.yaml

```diff
@@ -1824,58 +1824,146 @@ components:
         - $ref: '#/components/schemas/SetPropertiesUpdate'
         - $ref: '#/components/schemas/RemovePropertiesUpdate'
 
-    TableRequirement:
```

Comment:
Nit: Should TableRequirement be defined first?

Reply:
Yes, I don't recall this isn't the case.